src/main/java/eu/decentsoftware/holograms/api/listeners/PlayerListener.java

```diff
@@ -53,6 +64,89 @@ public void onRespawn(PlayerRespawnEvent e) {
         S.async(() -> decentHolograms.getHologramManager().hideAll(player));
     }
 
+
+    // 视角对着的全息
+    private Map<Player,String> viewHolograms = new ConcurrentHashMap<>();
+
+
+    private Map<Player,Long> tickLog = new ConcurrentHashMap<>();
+
+    public Map<Chunk,List<Hologram>> chunkListMap = new ConcurrentHashMap<>();
+
+    private void ref(){
+        chunkListMap.clear();
+        decentHolograms.getHologramManager().getHolograms().stream().forEach(hologram -> {
+            Chunk chunk = hologram.getLocation().getChunk();
+            chunkListMap.putIfAbsent(chunk,new ArrayList<>());
+            chunkListMap.get(chunk).add(hologram);
+        });
+    }
+
+
+    @EventHandler
+    public void onRegisterHd(DecentHologramsRegisterEvent registerEvent){
+        this.ref();
+    }
+
+    @EventHandler
+    public void onPlayerMove(PlayerMoveEvent playerMoveEvent){
+        Player player = playerMoveEvent.getPlayer();
+        Long orDefault = tickLog.getOrDefault(player, 0L);
+
+        if(chunkListMap.isEmpty()) ref();
+
+        if((System.currentTimeMillis()-orDefault) >= 200L){
+            Optional.ofNullable(chunkListMap.get(player.getLocation().getChunk())).ifPresent(list->{
+                List<Hologram> holograms = list.stream().collect(Collectors.toList());
```

Comment:
Is the created `holograms` list here even used?
Why exactly do you even do this if a list is returned already through the lsmbda?